client/app/bundles/course/survey/containers/ShowSurvey.jsx

```diff
@@ -131,11 +132,13 @@ class ShowSurvey extends React.Component {
     const { surveys, params: { courseId, surveyId } } = this.props;
     const survey = surveys && surveys.length > 0 ?
                    surveys.find(s => String(s.id) === String(surveyId)) : {};
+    const publishToggle =
```

Comment:
actually why not merge this with `SurveyDetails` (instead of passing as a variable), their PropTypes are almost same  ?